app/src/main/java/com/gulderbone/cookieclicker/cookieproducers/Grandma.kt

```diff
@@ -1,3 +1,3 @@
 package com.gulderbone.cookieclicker.cookieproducers
 
-class Grandma : CookieProducer("Grandma", 5, 50)
\ No newline at end of file
+class Grandma : CookieProducer("Grandma", 1, 100)
```

Comment:
I'm thinking about an enum or something like that to keep prices and cmp of all producers in one place.

Reply:
Not today, I'm too lazy. Added to backlog xd.